diff --git a/UAM-Toy-Env/Data/UAM Videos and Images/uam_toy.gif b/UAM-Toy-Env/Data/UAM Videos and Images/uam_toy.gif
index e69de29b..8ce64801 100644
Binary files a/UAM-Toy-Env/Data/UAM Videos and Images/uam_toy.gif and b/UAM-Toy-Env/Data/UAM Videos and Images/uam_toy.gif differ
diff --git a/UAM-Toy-Env/train.py b/UAM-Toy-Env/train.py
index 06d58b4c..830061a9 100644
--- a/UAM-Toy-Env/train.py
+++ b/UAM-Toy-Env/train.py
@@ -15,7 +15,7 @@ from wandb.integration.sb3 import WandbCallback
 sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 from uam_toy_environment.environ.uam_toy_environment import UAMToyEnvironment
 
-total_timesteps = 10e5  # Define the total timesteps for training.
+total_timesteps = 5e5  # Define the total timesteps for training.
 # Initialize WandB for logging.
 wandb.init(
     project="uam-toy-project",  # Change this to your project name
diff --git a/UAM-Toy-Env/uam_toy_environment/environ/uam_toy_environment.py b/UAM-Toy-Env/uam_toy_environment/environ/uam_toy_environment.py
index 61283866..589a1339 100644
--- a/UAM-Toy-Env/uam_toy_environment/environ/uam_toy_environment.py
+++ b/UAM-Toy-Env/uam_toy_environment/environ/uam_toy_environment.py
@@ -180,8 +180,8 @@ class UAMToyEnvironment(gym.Env):
             return {
                 "drone_pos": drone_pos,
                 "drone_vel": drone_vel,
-                "rel_drone_positions": rel_drone_positions,
-                "rel_drone_velocities": rel_drone_velocities,
+                # "rel_drone_positions": rel_drone_positions,
+                # "rel_drone_velocities": rel_drone_velocities,
                 "rel_obst_positions": rel_obst_positions,
                 # "rel_obst_velocities": rel_obst_velocities,
                 "rel_vert_positions": rel_vert_positions,
@@ -317,7 +317,7 @@ class UAMToyEnvironment(gym.Env):
         # reward = 1 * reward_goal + agent_collision + obstacle_collision + out_of_bounds \
         #  + approaching_vertiport + vertiport_reached
         
-        reward = 2 * reward_goal + out_of_bounds + approaching_vertiport
+        reward = reward_goal + out_of_bounds
 
         return reward
 
@@ -657,18 +657,18 @@ class UAMToyEnvironment(gym.Env):
                 shape=(2,),
                 dtype=np.float32,
             ),
-            "rel_drone_positions": Box(
-                low=-self.grid_size,
-                high=self.grid_size,
-                shape=(self.num_drones - 1, 2),  # For one drone, this would be shape (0, 2)
-                dtype=np.float32,
-            ),
-            "rel_drone_velocities": Box(
-                low=-max_rel_drone_vel,
-                high=max_rel_drone_vel,
-                shape=(self.num_drones - 1, 2),
-                dtype=np.float32,
-            ),
+            # "rel_drone_positions": Box(
+            #     low=-self.grid_size,
+            #     high=self.grid_size,
+            #     shape=(self.num_drones - 1, 2),  # For one drone, this would be shape (0, 2)
+            #     dtype=np.float32,
+            # ),
+            # "rel_drone_velocities": Box(
+            #     low=-max_rel_drone_vel,
+            #     high=max_rel_drone_vel,
+            #     shape=(self.num_drones - 1, 2),
+            #     dtype=np.float32,
+            # ),
             "rel_obst_positions": Box(
                 low=-self.grid_size,
                 high=self.grid_size,
diff --git a/uam_toy.zip b/uam_toy.zip
index 5823095b..da04d2cd 100644
Binary files a/uam_toy.zip and b/uam_toy.zip differ
diff --git a/wandb/latest-run b/wandb/latest-run
index 38beb94a..63e403d0 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20250614_045941-n16psgiw
\ No newline at end of file
+run-20250709_004445-7dj4roz4
\ No newline at end of file
